Tests/TezosKit/MichelsonTests.swift

```diff
@@ -0,0 +1,238 @@
+// Copyright Keefer Taylor, 2019
+
+@testable import TezosKit
+import XCTest
+
+// swiftlint:disable force_cast
+// swiftlint:disable force_try
+// swiftlint:disable line_length
+
+final class MichelsonTests: XCTestCase {
+  static let michelsonUnit = UnitMichelsonParameter()
+  static let expectedMichelsonUnitEncoding = "{\"prim\":\"unit\"}"
+
+  static let michelsonString = StringMichelsonParameter(string: "tezoskit")
+  static let expectedMichelsonStringEncoding = "{\"string\":\"tezoskit\"}"
+
+  static let michelsonInt = IntMichelsonParameter(int: 42)
+  static let expectedMichelsonIntEncoding = "{\"int\":\"42\"}"
+
+  static let michelsonPair = PairMichelsonParameter(left: michelsonString, right: michelsonInt)
+  static let expectedMichelsonPairEncoding =
+    "{\"prim\":\"Pair\",\"args\":[\(expectedMichelsonStringEncoding),\(expectedMichelsonIntEncoding)]}"
+
+  static let michelsonLeft = LeftMichelsonParameter(arg: michelsonString)
+  static let expectedMichelsonLeftEncoding = "{\"prim\":\"Left\",\"args\":[\(expectedMichelsonStringEncoding)]}"
+
+  static let michelsonRight = RightMichelsonParameter(arg: michelsonInt)
+  static let expectedMichelsonRightEncoding = "{\"prim\":\"Right\",\"args\":[\(expectedMichelsonIntEncoding)]}"
+
+  static let michelsonTrue = BoolMichelsonParameter(bool: true)
+  static let expectedMichelsonTrueEncoding = "{\"prim\":\"true\"}"
+
+  static let michelsonFalse = BoolMichelsonParameter(bool: false)
+  static let expectedMichelsonFalseEncoding = "{\"prim\":\"false\"}"
+
+  static let michelsonBytes = BytesMichelsonParameter(hex: "deadbeef")
+  static let expectedMichelsonBytesEncoding = "{\"bytes\":\"deadbeef\"}"
+
+  static let michelsonSome = SomeMichelsonParameter(some: michelsonInt)
+  static let expectedMichelsonSomeEncoding = "{\"prim\":\"some\",\"args\":[\(expectedMichelsonIntEncoding)]}"
```

Comment:
I believe these are case-sensitive, so it would be "Some", not "some"; same form None, Left, Right, True, False, Unit.